pkg/logs/input/kubernetes/convertor.go

```diff
@@ -12,6 +12,21 @@ import (
 	"regexp"
 )
 
+var (
+	// 2019-05-29T13:27:27.482052544Z
+	timestampMatcher = regexp.MustCompile("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{9}Z")
+	statusMatcher    = regexp.MustCompile("std(out|err)")
```

Comment:
Do we really need a regex for this simple check ? What's the overhead compare to doing a simple condition ?